nucypher/characters/lawful.py

```diff
@@ -438,7 +438,9 @@ def decrypt_message_kit(self, label: bytes, message_kit: MessageKit) -> List[byt
 
 class Bob(Character):
     banner = BOB_BANNER
+    default_dkg_variant = FerveoVariant.PRECOMPUTED
```

Comment:
@piotr-roslaniec is `precomputed` supposed to be the default or `simple`. I thought `simple` was the one with an actual threshold < num_shares, and for `precomputed` threshold == num_shares.

I keep mixing up the two 😓 ...

Reply:
Both should work just fine. It's [my humble opinion](https://discord.com/channels/866378471868727316/866378471868727319/1125366637592313916) that we should `precomputed` as a default, but if there is a preference for more robustness (threshold < num_shares) then there is nothing stopping us from using `simple` variant. We just need to make sure that the decrypted understands which variant is being used to collect the necessary amount of decryption shares.

Reply:
> It's [my humble opinion](https://discord.com/channels/866378471868727316/866378471868727319/1125366637592313916)

The issue is not the ritual really (yes that requires all nodes to participate), but rather on the decryption side i.e. Bob. If I understand correctly, `precomputed` requires all `n` decryption fragments to be obtained before decryption vs `simple` which requires a threshold of them i.e. only `m`-of-`n` where `m` < `n`.

`precomputed` therefore, can lead to availability issues i.e. if any node wasn't properly up and running. Only requiring a threshold is one of the main tenets of our fundamental value prop for CBD. For example, the parameter package of 16-of-30 or whatever it was would not be possible with `precomputed`. 

If my understanding is correct, then I think `simple` should be the default, and if some adopter wants to risk availability by using `precomputed` they can do so if they like. (cc @cygnusv , @arjunhassard )

Reply:
I agree that `simple` makes more sense as the default.

We also need to bury it better - it needs to be modifiable only with great intent.